src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -96,7 +110,7 @@ const ChannelTitle = ({
 
           <View style={styles.chatMessage}>
             <Text numberOfLines={1} ellipsizeMode="tail" style={styles.chatContentMessage}>
-              {`${isMe ? 'You: ' : ''}${message}`}
+              {getMessageText()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `getMessageText()` function here is appropriate as it allows for dynamic rendering of the message text based on whether there's an attachment or not.
```diff
- {`${isMe ? 'You: ' : ''}${message}`}
+ {getMessageText()}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->